docker/lib/dependabot/docker/metadata_finder.rb

```diff
@@ -2,14 +2,28 @@
 
 require "dependabot/metadata_finders"
 require "dependabot/metadata_finders/base"
+require "dependabot/shared_helpers"
 
 module Dependabot
   module Docker
     class MetadataFinder < Dependabot::MetadataFinders::Base
       private
 
       def look_up_source
-        # TODO: Find a way to add links to PRs
+        return if dependency.requirements.empty?
+
+        new_source = dependency.requirements.first[:source]
+        return unless new_source && new_source[:registry] && new_source[:tag]
+
+        image_ref = "#{new_source[:registry]}/#{dependency.name}:#{new_source[:tag]}"
+        image_details_output = SharedHelpers.run_shell_command("regctl image inspect #{image_ref}")
+        image_details = JSON.parse(image_details_output)
+        image_source = image_details.dig("config", "Labels", "org.opencontainers.image.source")
+        return unless image_source
+
+        Dependabot::Source.from_url(image_source)
```

Comment:
> It is also possible for the OCI annotations to indicate the commit associated with a docker image

Should we additionally check whether the source points to a release and not a commit?
I'm unsure of whether a PR would get created or fail if the metadata finder encounters an error

Reply:
The source here is just a link to the repo. Figuring out the releases/commits comes from the dependency requirements that the update checker produced.

I don't have a test case for when the tags aren't present on the repo since that's handled elsewhere but I saw in testing that the PR includes a link to the repo and you don't get any of the change details.